### PR TITLE
Ruined COC Relay Station

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds a new asteroid ruin. It's a COC relay station. Map done by steviii so if there are any issues it's their fault.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by DreamySkrell** [comment]
The mapdiffbot2 is really dumb lately - please show some screenshots of this map.

**Comment by RustingWithYou** [comment]
> The mapdiffbot2 is really dumb lately - please show some screenshots of this map.

image

File: maps/random_ruins/exoplanets/asteroid/coalition_base/coalition_base.dm [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/datum/map_template/ruin/exoplanet/coalition_base
+    name = "Abandoned Coalition Relay"
+    id = "coalition_base"
+    description = "hehehe cock base"
```
Comment (DreamySkrell) [inline maps/random_ruins/exoplanets/asteroid/coalition_base/coalition_base.dm:4]: hehehe cock base

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

File: maps/random_ruins/exoplanets/asteroid/coalition_base/coalition_base.dm (hunk 1 above)
Comment (RustingWithYou) [reply maps/random_ruins/exoplanets/asteroid/coalition_base/coalition_base.dm:4]: fuck i forgot to change this

**Comment by TheDanseMacabre** [comment]
could be worth using random gun spawners instead of spawning the same guns every time, the civilian gun spawner could be a good option

**Review by DreamySkrell — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**